src/app/collection/_components/Map/MapModal.js

```
@@ -0,0 +1,140 @@
+'use client';
+
+import { createPortal } from 'react-dom';
+import { FaXmark } from 'react-icons/fa6';
+import { Map, Polygon, MapMarker } from 'react-kakao-maps-sdk';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)